File: examples-sdk/go/405-agent-react-multi-tool/main.go
// 405-agent-react-multi-tool — Go SDK equivalent
// Compile: vil compile --from go --input 405-agent-react-multi-tool/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("react-multi-tool-agent", 3124)

	react_agent := vil.NewService("react-agent")
	react_agent.Endpoint("POST", "/react", "react_handler")
	s.Service(react_agent)

	s.Compile()
}

File: examples-sdk/zig/405-agent-react-multi-tool/main.zig
// 405-agent-react-multi-tool — Zig SDK equivalent
// Compile: vil compile --from zig --input 405-agent-react-multi-tool/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("react-multi-tool-agent", 3124);
    var react_agent = vil.Service.init("react-agent");
    react_agent.endpoint("POST", "/react", "react_handler");
    server.service(&react_agent);
    server.compile();
}
